doc/benchmark_workflow/visualize_benchmark.rst

```diff
@@ -0,0 +1,74 @@
+.. _visualize_benchmark:
+
+Visualize a benchmark
+=====================
+
+After running a benchmark, benchopt automatically displays the results in an HTML dashboard..
+
+Let's explore the dashboard features on the Lasos benchmark.
+
+.. Hint::
+
+    Head to :ref:`get_started` to learn how to install benchopt
+    and setup the Lasso benchmark accordingly.
+
+.. figure:: ../_static/annotated_benchmark_dashboard.png
+   :align: center
+   :alt: Dashboard of the Lasso benchmark results
+
+
+Part 1: Header
+--------------
+
+Here are the metadata of the benchmark - namely the benchmark title
+and the specifications of the machine used to run it.
+On the right hand side is a :kbd:`Download` button to download the output of the benchmark as ``.parquet`` file.
+
+Part 2: Figure
+--------------
+
+Here is the main figure that plots the tracked metrics throughout the benchmark run.
+Its title shows the objective and the dataset names and their corresponding parameters that produced the plot.
+
+Hover over the figure will make a modebar appear in the right side.
+This can be used to interact with the figure, e.g. zoom in and out on particular regions.
+
+Part 3: Legend
+--------------
+
+The legend maps every curve to a solver.
+
+Click a legend item to hide/show its corresponding solver.
+Similarly, double-click a legend item to hide/show all the other solvers.
+Finally, hovering over a legend item shows a tooltip with details about the solver.
+
+.. note::
+
+    Displayed details about the solver are the content of the solver's docstring, if present.
+
+Part 4: Sidebar
+---------------
+
+The first two dropdown menus, **Data** and **Objective**, enable to select a benchmark setup.
+The **Data** dropdown contains all the datasets included in the benchmark as well as their parameters.
+The same as for **Objective** dropdown.
+
+The **Objective_column** exposes all the tracked metrics along the benchmark run.
```

Comment:
no underscore

Reply:
btw maybe Objective column is not the clearest name, "metric" may be more informative? wdyt?

Reply:
To avoid confusion, we better use the name in the dashboard, **Objective column**?

Reply:
I'm incidentally suggesting to rename it in the dashboard too. It's not really clear what "column" refers to here

Reply:
I opened a PR #633 to make this change.